custom/icds_reports/utils/aggregation_helpers/agg_ccs_record.py

```diff
@@ -89,9 +89,9 @@ def aggregation_query(self):
             ('valid_visits', 'sum(crm.valid_visits)'),
             ('expected_visits', 'sum( '
              'CASE '
-             'WHEN ucr.pregnant=1 THEN 0.44 '
-             'WHEN ucr.month - ucr.add <= 0 THEN 6 '
-             'WHEN ucr.month - ucr.add < 182 THEN 1 '
+             'WHEN crm.pregnant=1 THEN 0.44 '
+             'WHEN crm.month - crm.add <= 0 THEN 6 '
+             'WHEN crm.month - crm.add < 182 THEN 1 '
              'ELSE 0.39 END'
```

Comment:
This has increased the expected_visits in the output. Why is that?

Reply:
yeah this is because, earlier no ucr.add value was there because not fixture was there therefore it was going to the last case i.e `ELSE 0.39 END` for all lady which are not pregnant.
Now since the fixture is added therefore  `expected_visits` is changed. @emord